### ISSUE TITLE
ReactCrop capturing scroll events on mobile

### ISSUE THREAD

**Issue by NathanC** [dominictobias/react-image-crop#494]
Hello! I have a large vertical image in my `ReactCrop` that take up the full width and height on mobile. I have a 1/1 aspect ratio crop, to select a square portion of the vertical image.

![image](https://user-images.githubusercontent.com/6182419/173628900-a2ef7e07-5a10-4974-88fd-77eb7fa1ec22.png)

The user is unable to scroll past this image, because the scroll gesture is captured by `ReactCrop`. So the user gets stuck in this part of the form. When the user tries to scroll by gesturing on the image (outside of the crop), nothing happens, and the containing div is not scrolled.

The behavior I would like is that, if the user gestures on the crop itself, the crop is moved. However, if the user gestures on the blank space in the image **outside of the crop**, it should scroll past the image.

I'm guessing what's happening is touch or gesture scroll events (not sure how it's handled on mobile) are being caught and not allowed to bubble. Would it be possible to check if the user is pressing in the bounding box of the crop, and bubble the scroll event if the user it outside of the crop?

Thanks in advance!

**Comment by NathanC** [comment]
Alright, post is ready now.

**Comment by rosareyes** [comment]
Is there any news on this? I'm currently having the same issue

**Comment by DominicTobias** [comment]
Not sure how this would be fixed since the crop has to capture input in order to draw. It would also be a bug to scroll as the user is trying to draw a crop (and there have been bugs about that a long time ago).

I would suggest a fullscreen experience for mobile (this is what apps do like Photos, Whatsapp and all others). Or have an edit mode and if the user isn't in edit mode then put a div (could be transparent) over the top of the crop.

**Comment by rosareyes** [comment]
Exactly, that's what I did, and I have set `disabled` to `true`, so the user can only click on the squares generated but not modify them, but in this scenario, is there a way of having `disabled = true`, and the user could scroll down and still being able to click on the square to enter edit mode? 

For example here, `disabled = true`, and if we tap the crop square, we enter edit mode, but we cannot scroll.

Thank you in advanced.
![photo_2022-11-22_22-52-53](https://user-images.githubusercontent.com/63470281/203519696-60202c6e-4e15-40e3-bbeb-07dfb31790e6.jpg)

**Comment by DominicTobias** [comment]
> Exactly, that's what I did, and I have set `disabled` to `true`, so the user can only click on the squares generated but not modify them, but in this scenario, is there a way of having `disabled = true`, and the user could scroll down and still being able to click on the square to enter edit mode?
> 
> For example here, `disabled = true`, and if we tap the crop square, we enter edit mode, but we cannot scroll.
> 
> Thank you in advanced. ![photo_2022-11-22_22-52-53](https://user-images.githubusercontent.com/63470281/203519696-60202c6e-4e15-40e3-bbeb-07dfb31790e6.jpg)

Ah I see, in this case it sounds like a bug and disabled = true should allow scroll, I will fix that

**Comment by DominicTobias** [comment]
> Exactly, that's what I did, and I have set `disabled` to `true`, so the user can only click on the squares generated but not modify them, but in this scenario, is there a way of having `disabled = true`, and the user could scroll down and still being able to click on the square to enter edit mode?
> 
> For example here, `disabled = true`, and if we tap the crop square, we enter edit mode, but we cannot scroll.
> 

Please try 10.0.9, should be able to scroll with disabled=true
